### ISSUE TITLE
bash map_lariats.sh failed

### ISSUE THREAD

**Issue by santataRU** [jlbuerer/LaMIRA#9]
Dear Developer,

I downloaded the code and tried to run it but failed with the message below:
Command line:
bash map_lariats.sh /Users/bieniaszlab/Documents/Xiao/Peng_Zhang/test_examples/5ss_mut/5ss_test_WT.fq 197 /Users/bieniaszlab/Documents/Xiao/CLIP/index/hg38/bowtieindex/hg38_genome /Users/bieniaszlab/Documents/Xiao/CLIP/index/hg38/Homo_sapiens.GRCh38.108.sorted.gtf  1 ~/Desktop

Error message:
**python3: can't open file '/Users/bieniaszlab/Downloads/LaMIRA-master/./mapping_scripts/create_SS_and_intron_tables.py': [Errno 2] No such file or directory
Can't locate aligner.pm in @INC (you may need to install the aligner module) (@INC contains: ./ /Users/bieniaszlab/opt/miniconda3/lib/site_perl/5.26.2/darwin-thread-multi-2level /Users/bieniaszlab/opt/miniconda3/lib/site_perl/5.26.2 /Users/bieniaszlab/opt/miniconda3/lib/5.26.2/darwin-thread-multi-2level /Users/bieniaszlab/opt/miniconda3/lib/5.26.2 .) at ./mapping_scripts/find_lariats.pl line 7.
BEGIN failed--compilation aborted at ./mapping_scripts/find_lariats.pl line 7.**

I appreciate it if there is any input.

Regards,

Xiao

**Comment by santataRU** [comment]
map_lariats.sh line 39 `python3 ./mapping_scripts/create_SS_and_intron_tables.py` should be `python3 ./mapping_scripts/create_SS_and_intron_table.py`. This causes "[Errno 2] No such file or directory.

**Comment by jlbuerer** [comment]
Yes, sorry there appears to be an error with the relative paths here. However, I do not maintain this version of the lariat mapping code. I would suggest you use [lariat_mapping](https://github.com/fairbrother-lab/lariat_mapping) instead

**Comment by santataRU** [comment]
Thank you for the suggestions!
